lib/gds_api/email_alert_api.rb

```diff
@@ -100,11 +100,12 @@ def unsubscribe(uuid)
   # Subscribe
   #
   # @return [Hash] subscription_id
-  def subscribe(subscribable_id:, address:)
+  def subscribe(subscribable_id:, address:, frequency:)
```

Comment:
Should we make it default to `immediately`? (Then this doesn't have to be a breaking change)

Reply:
yep - fixed in 2f59e73